### PR TITLE
#702 update ldf config

### PR DESCRIPTION
**Pull request by roefem**

### CODE REVIEW

File: development.ini [hunk 1]
```diff
@@ -47,7 +47,7 @@ atramhasis.session_factory.secret = itsaseekreet
 atramhasis.dump_location = %(here)s/datadumps
 
 # Location of rdf2hdt executable
-atramhasis.rdf2hdt = /link/to/rdf2hdt
+atramhasis.rdf2hdt = /usr/local/bin/rdf2hdt
```
Comment (goessebr) [inline development.ini:50]: I doubt if we want to change this. This repo can be downloaded by users using all kind of machines. `/usr/local/bin/rdf2hdt` makes the suggestion that the file will always be on that location. On f.e. Windows machines that won't be the case. `/link/to` is more clear that this is a setting that the user will need to change.

Or we could use `atramhasis.rdf2hdt = /usr/local/bin/rdf2hdt` and adjust the comment above, something like `# Location of rdf2hdt executable. Change this location to the path where the rdf2hdt can be found on your machine`

**Review by goessebr — CHANGES_REQUESTED** [review]
Looks fine, output is like asked. 1 remark about the ini config.

**Review by goessebr — APPROVED** [review]

**Comment by goessebr** [comment]
Ik zie het nu pas maar er staan nog 2 TODO's in de changes, dient daar nog iets te gebeuren?

File: atramhasis/scripts/generate_ldf_config.py [hunk 2]
```diff
@@ -44,102 +43,104 @@ def main():
...
-            'title': 'All conceptschemes',
-            'type': 'CompositeDatasource',
-            'description': 'All conceptschemes contained in this Atramhasis instance together.',
-            'settings': {
-                'references': pids
-            }
+        composite_sourceconfig = {
+            "@id": "urn:ldf-server:myDatasourcecomposite",
+            "@type": "CompositeDatasource",
+            "quads": False,  # TODO
```
Comment (roefem) [inline atramhasis/scripts/generate_ldf_config.py:127]: @koenedaele @goessebr 
the [migration script](https://github.com/LinkedDataFragments/Server.js/wiki/Release-3.0.0#migration-from-2xx) explicitly set this value to False, so I did so as well here. Should quad patterns be supported? If this field is omitted from the config, it defaults to True

**Pull request merged**